ertviz/ert_client/__init__.py

```diff
@@ -1,8 +1,8 @@
 from ertviz.models import Response, EnsembleModel
 
 
-def get_response(ref_url):
-    return Response(ref_url=ref_url)
+def get_response(name, ref_url):
```

Comment:
Is this function even used somewhere?

Reply:
Maybe not

Reply:
Then we can remove it (from tests too I guess).

Reply:
done!